tuxemon/core/states/dialog/__init__.py

```diff
@@ -60,3 +63,29 @@ def next_text(self):
             return text
         except IndexError:
             return None
+
```

Comment:
To animate the avitar, pass apyganim object as the avatar.  Then create a function hook here called "draw":
```python
def draw(self, surface):
   super(DialogState, self).draw(surface)
   # self.avatar.blit(...)
```

Reply:
I resolved the animation part by creating a new load_animated_sprite function in tools.py: It can be reused for future animations, and also serves to spare others of having to rediscover what I spent a day finding out... mostly the same as load_sprite except it takes a list of images and a delay then returns an animated surface instead. That draw function was not even necessary, as issuing animation.play() when creating the sprite activates it just fine.